### ISSUE TITLE
Cannot paste into commit message

### ISSUE THREAD

**Issue by tfnico** [codingtwinky/atom-ungit#25]
Atom seems to be capturing the Paste command. This kind of ruins
the experience of committing with Ungit.

**Comment by codingtwinky** [comment]
I did some digging, and when you type in `cmd-v` in keybinding view, one can see that there are multiple entries.  For example, tree-view has it's own implementation of cmd-v.  Which is understandable.

So it seems that ungit would have to be implemented ins a similar fashion, especially considering ungit is designed after https://atom.io/packages/atom-html-preview and it is suffering same problem.  

Which is supper annoying for an webview like this one.  


In short, I think this will be a tricky one despite it being rudimentary and basic functionality.

**Comment by seleckis** [comment]
+1
And cannot paste into "Enter path to repository" field.

**Comment by xHN35RQ** [comment]
Works for me Atom 1.1.0, atom-ungit 0.5.2
